netebpfext/user/fwp_um.cpp

```diff
@@ -852,6 +874,7 @@ _IRQL_requires_min_(PASSIVE_LEVEL) _IRQL_requires_max_(DISPATCH_LEVEL) FWPS_CONN
     FwpsQueryConnectionRedirectState0(
         _In_ HANDLE redirectRecords, _In_ HANDLE redirectHandle, _Outptr_opt_result_maybenull_ void** redirectContext)
 {
+    // Skip fault injection as the return is FWPS_CONNECTION_NOT_REDIRECTED.
```

Comment:
Maybe we can return `FWPS_CONNECTION_REDIRECTED_BY_SELF` or `FWPS_CONNECTION_PREVIOUSLY_REDIRECTED_BY_SELF` when under fault injection and that will help trigger different code path in netebpfext?

@shankarseal do you think it will be helpful?

Reply:
Good suggestion. I'll add it to the PR#2424